README.md

```diff
@@ -21,7 +21,33 @@ You will need the following things properly installed on your computer.
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 * Visit your tests at [http://localhost:4200/tests](http://localhost:4200/tests).
 
-## tests
+## Tests
 
-* dont forget to run cypress test locally: npm run cypress-local
+There are multiple make commands ready for you.
+Check the file `Makefile` for specifics.
+
+### Auto test Requirements
+To make these tests work you will need checkout of the [project](https://github.com/kanselarij-vlaanderen/kaleidos-project) next to the frontend folder.
+There you will need some customsations in your docker-compose.override.yml
```

Comment:
New people will be puzzled about this. Provide small context about the exact origin of those customisations?

Reply:
Added that in.